NewForce-Cohort-9/newforce-c9-tabloid-full-stack-TabloidFullStack
# Subscribed Posts on Homepage

# What changed?

Posts from users you are subscribed to will now appear on the homepage.

# How to test?

1. Add and commit your files
2. git checkout main
3. git fetch --all
4. git checkout tylerBranch-subscribedPostsOnHome
5. git pull origin tylerBranch-subscribedPostsOnHome
6. Run your API and app
7. Log in as foo@bar.com (or whoever you subscribed to a user with earlier.)
8. You should be directed straight to a page where you see all of the posts you are subscribed to, in detail.
9. Click on "Posts" and then back on "Home" and it should take you back to the same page.
10. Congratulations, you have viewed what you are subscribed to.
